.github/FUNDING.yml

```diff
@@ -0,0 +1,3 @@
+# These are supported funding model platforms
```

Comment:
This seems like a line from a `FUNDING.yml` template :thinking: I think we can either remove it, or if we want this file to make sense outside of GitHub (which I think is unnecessary as it is in a folder called `.github`) make it a bit more descriptive.

Reply:
It is indeed from a template, happy to remove it 👌